spec/controllers/idv/document_capture_controller_spec.rb

```diff
@@ -106,6 +113,10 @@
         selfie_check_required: sp_selfie_enabled,
       }.merge(ab_test_args)
     end
+    it 'has non-nil presenter' do
```

Comment:
Very minor, but to keep things consistent you need to add a newline above the `it` block.

```suggestion
    end
    
    it 'has non-nil presenter' do
```